File: suma.py
"""
Módulo para calcular la suma de 2 números
"""

def sumar_numeros(num1, num2):
    """
    Calcula la suma de dos números
    
    Args:
        num1: Primer número
        num2: Segundo número
        
    Returns:
        float: La suma de los dos números
    """
    return num1 + num2


def main():
    """Función principal para calcular suma"""
    print("\n" + "=" * 40)
    print("CALCULADORA DE SUMA")
    print("=" * 40)
    
    try:
        num1 = float(input("Ingrese el primer número: "))
        num2 = float(input("Ingrese el segundo número: "))
        
        resultado = sumar_numeros(num1, num2)
        
        print(f"\n{num1} + {num2} = {resultado}")
        print("=" * 40 + "\n")
        
    except ValueError:
        print("Error: Ingrese números válidos")


if __name__ == "__main__":
    main()

File: suma.php
<?php
?>
<!DOCTYPE html>
<html>
<head>
    <title>Calcular Suma</title>
</head>
<body>
    <h1>Calculadora de Suma</h1>
    
    <form method="POST">
        <label>Primer Número:</label>
        <input type="number" name="num1" step="any" required>
        <br><br>
        
        <label>Segundo Número:</label>
        <input type="number" name="num2" step="any" required>
        <br><br>
        
        <button type="submit">Calcular</button>
    </form>
    
    <?php
    if ($_SERVER['REQUEST_METHOD'] == 'POST') {
        $num1 = $_POST['num1'];
        $num2 = $_POST['num2'];
        $suma = $num1 + $num2;
        
        echo "<h2>Resultado: $num1 + $num2 = $suma</h2>";
    }
    ?>
    
    <br>
    <a href="index.php">Volver al inicio</a>
</body>
</html>
